R/article.R

```diff
@@ -88,9 +88,9 @@ amq_article <- function(
 #'   <https://www.ametsoc.org/ams/index.cfm/publications/authors/journal-and-bams-authors/author-resources/latex-author-info/>.
 #' @export
 #' @rdname article
-ams_article <- function(..., keep_tex = TRUE, md_extensions = c("-autolink_bare_uris")) {
+ams_article <- function(..., keep_tex = TRUE, md_extensions = c("-autolink_bare_uris", "-auto_identifiers")) {
   pdf_document_format(
```

Comment:
We need to put a Pandoc requirement before running the function here - the test are failing because of this mainly
```
rmarkdown::pandoc_available('2.10', TRUE)
````